### PR TITLE
pushing user tables

### PR DESCRIPTION
**Pull request by dolfo9**

### CODE REVIEW

File: db/init_db.js [hunk 1]
```diff
@@ -11,6 +11,19 @@ async function buildTables() {
     // drop tables in correct order
 
     // build tables in correct order
+    await client.query(`
+      CREATE TABLE users (
+        id SERIAL PRIMARY key,
+        firstName VARCHAR(255) NOT NULL,
+        lastName TEXT NOT NULL,
+        email VARCAHAR(320) NOT NULL,
```
Comment (samrunyan5) [inline db/init_db.js]: typo in VARCHAR needs to be corrected

File: db/init_db.js [hunk 2]
```diff
@@ -11,6 +11,19 @@ async function buildTables() {
...
+        image VARCHAR(255) UNIQUE,
```
Comment (samrunyan5) [inline db/init_db.js]: image should be imageURL
this also needs a default value, you can do something like `imageURL text default 'someone@gmail.com'`

**Review by samrunyan5 — DISMISSED** [review]
Added some comments in the PR for changes that need to be made!

**Review by crystaljoyce — APPROVED** [review]
Thanks for updating! No obvious errors.

**Comment by crystaljoyce** [comment]
This looks good.

**Review by crystaljoyce — APPROVED** [review]
These look good.

**Review by crystaljoyce — APPROVED** [review]
These look good!

**Pull request merged**